src/neovim/neovim.ts

```diff
@@ -30,7 +30,8 @@ export class Neovim implements vscode.Disposable {
     // Execute the command
     logger.debug(`Neovim: Running ${command}.`);
     await this.nvim.input(command);
-    if ((await this.nvim.getMode()).blocking) {
+    const mode = await this.nvim.mode;
```

Comment:
@justinmk this is not returning, any ideas? Also any advice in general for this PR would be great

Reply:
call `nvim_ui_attach` after connecting. Nvim waits until then before continuing, so that UI messages can be received.

Reply:
Got it, thanks! Since I am only partially using the ui features, I am specifying a bogus width/height... seems bad but works for now..